packages/developer-console-spa/src/components/AddAppForm.tsx

```diff
@@ -9,13 +9,11 @@ import * as yup from 'yup';
 
 interface IAppInput {
   name: string;
-  path: string;
   description: string;
 }
 const appSchema = yup.object().shape( {
-  name: yup.string().required(),
-  path: yup.string().required(),
-  description: yup.string(),
+  name: yup.string().trim().required(),
+  description: yup.string().trim(),
 } );
 
 export default function AppAppForm () {
```

Comment:
Don't you think this name is confusing...App App.. 😅

Reply:
Oh my bad. That was a typo, it should've been `AddAppForm`. Fixed it.